README.md

```diff
@@ -40,98 +39,60 @@ The following section explains how to use the Gonebusy library in a new project.
-
-Add a PHP file to this project
-
-![Add a new project in PHPStorm - Step 3](http://apidocs.io/illustration/php?step=createFile&workspaceFolder=Gonebusy-PHP)
+Create a new directory. Name the directory as "test".
 
-Name it "testSDK"
+Add a PHP file to this project. Name it "testSDK.php".
 
-![Add a new project in PHPStorm - Step 4](http://apidocs.io/illustration/php?step=nameFile&workspaceFolder=Gonebusy-PHP)
+Depending on your project setup, you might need to include composer's autoloader in your PHP code to enable auto loading of classes"
 
-Depending on your project setup, you might need to include composer's autoloader in your PHP code to enable auto loading of classes.
-
-```PHP
+```php
 require_once "../vendor/autoload.php";
 ```
 
-It is important that the path inside require_once correctly points to the file ```autoload.php``` inside the vendor directory created during dependency installations.
-
-![Add a new project in PHPStorm - Step 4](http://apidocs.io/illustration/php?step=projectFiles&workspaceFolder=Gonebusy-PHP)
+> Note: the `../` path assumes you'll run testSDK.php directly from test/ .
 
 After this you can add code to initialize the client library and acquire the instance of a Controller class. Sample code to initialize the client library and using controller methods is given in the subsequent sections.
 
 ### 3. Run the Test Project
 
-To run your project you must set the Interpreter for your project. Interpreter is the PHP engine installed on your computer.
-
-Open ```Settings``` from ```File``` menu.
-
-![Run Test Project - Step 1](http://apidocs.io/illustration/php?step=openSettings&workspaceFolder=Gonebusy-PHP)
-
-Select ```PHP``` from within ```Languages & Frameworks```
-
-![Run Test Project - Step 2](http://apidocs.io/illustration/php?step=setInterpreter0&workspaceFolder=Gonebusy-PHP)
-
-Browse for Interpreters near the ```Interpreter``` option and choose your interpreter.
-
-![Run Test Project - Step 3](http://apidocs.io/illustration/php?step=setInterpreter1&workspaceFolder=Gonebusy-PHP)
-
-Once the interpreter is selected, click ```OK```
-
-![Run Test Project - Step 4](http://apidocs.io/illustration/php?step=setInterpreter2&workspaceFolder=Gonebusy-PHP)
-
-To run your project, right click on your PHP file inside your Test project and click on ```Run```
-
-![Run Test Project - Step 5](http://apidocs.io/illustration/php?step=runProject&workspaceFolder=Gonebusy-PHP)
+```sh
+cd test
+php testSDK.php
+```
 
 ## How to Test
 
 Unit tests in this SDK can be run using PHPUnit.
 
 1. First install the dependencies using composer including the `require-dev` dependencies.
-2. Run `vendor\bin\phpunit --verbose` from commandline to execute tests. If you have
+1. Run `vendor\bin\phpunit --verbose` from command line to execute tests. If you have
    installed PHPUnit globally, run tests using `phpunit --verbose` instead.
```

Comment:
If user has done `cd test`, won't this need to be `../vendor/bin/phpunit` ?

And yes, let's make sure we're using Unix `/` instead of Windows `\`.